src/algebra/module/basic.lean

```diff
@@ -119,6 +119,10 @@ variables {R M}
 @[simp] lemma smul_add_hom_apply (r : R) (x : M) :
   smul_add_hom R M r x = r • x := rfl
 
+@[simp] lemma smul_add_hom_one {R M : Type*} [semiring R] [add_comm_monoid M] [module R M] :
+  smul_add_hom R M 1 = add_monoid_hom.id _ :=
+by { ext, rw [smul_add_hom_apply, one_smul, add_monoid_hom.id_apply] }
```

Comment:
It's probably worth stating this for `const_smul_hom` too (and using that proof to prove this one).

It won't impact this PR, but as someone who only just found `smul_add_hom`, your opinion on [this Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Definition.20elimination.20contest/near/237347199) would be handy.

Reply:
Thanks, done